tests/test_sync.py

```diff
@@ -338,3 +372,63 @@ def test_synchronize_with_cleanup_cli():
     assert "src/ansys/tools/repo_sync/__init__.py" in captured
     assert "src/ansys/tools/repo_sync/__main__.py" in captured
     assert "src/ansys/tools/repo_sync/repo_sync.py" in captured
+
+
+@pytest.mark.skipif(SKIP_LOCALLY, reason="Only runs on workflow")
+def test_synchronize_with_no_sync_cli():
+    """Test synchronization tool (with no files needed to be synced) from CLI."""
+
+    # Define a temp directory and copy assets in it
+    temp_dir = tempfile.TemporaryDirectory(prefix="repo_clone_cli_")
+    shutil.copytree(
+        ASSETS_DIRECTORY,
+        temp_dir.name,
+        dirs_exist_ok=True,
+    )
+
+    # Requires installing project
+    subprocess.run(
+        [
+            "pip",
+            "install",
+            ".",
+        ],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        cwd=ROOT_PATH,
+    )
+
+    # Call CLI tool
+    completed_process = subprocess.run(
+        [
+            "repo-sync",
+            "--token",
+            TOKEN,
+            "--owner",
+            "ansys",
+            "--repository",
+            "ansys-tools-repo-sync",
+            "--from-dir",
+            "ansys",
+            "--to-dir",
+            "src/ansys",
+            "--include-manifest",
+            "manifest_no_files.txt",
+            "--skip-ci",
+            "--random-branch-name",
+        ],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        cwd=temp_dir.name,
+    )
+
+    # Check output info
+    print(completed_process.returncode)
+    print(completed_process.stdout)
+    print(completed_process.stderr)

```

Comment:
Why these prints in the tests?

Reply:
So that the output of the CLI tool is shown on screen on the workflow logs. Since we are using ``subprocess``, its output is not shown on screen. That way we can easily debug if something went wrong.